cmd/errtrace/main.go

```diff
@@ -1208,3 +1208,13 @@ func nVars(prefix string, n int) []string {
 	}
 	return vars
 }
+
+func logln(w io.Writer, s string) {
+	//nolint:errcheck // logging writes are best-effort
+	fmt.Fprintln(w, s)
```

Comment:
Ignoring with `_` is nicer than nolint, IMO

```suggestion
	_, _ = fmt.Fprintln(w, s) // logging writes are best-effort
```